arci-ros/src/rosrust_utils.rs

```diff
@@ -262,3 +266,34 @@ macro_rules! define_action_client {
         );
     };
 }
+
```

Comment:
I want to know what is this doing, and *why* it is needed (especially the if-else part). Could you add some documents for these functions?